worlds/sims4/Items.py

```
@@ -24,21 +25,29 @@ class Sims4Item(Item):
 
 skills_table: dict[int, ItemDict] = {}
 
+
+class SkillConfigError(ValueError):
+    """Raised when skill configuration is invalid."""
+    pass
```

Ruff (0.14.10)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)